xdp/src/packet.rs

```diff
@@ -0,0 +1,116 @@
+#![allow(clippy::arithmetic_side_effects)]
```

Comment:
not nice to have potentially faulty overflows in network parsing code.

Reply:
Can you point out to the functions that _parse_ data? And exactly to _which_ operations can overflow? 😉

I didn't slap that allow lint there lightly. This code is in the hot path, and none of the operations can overflow.

Reply:
Maybe slap the allow on individual functions rather than whole file?

The performance of `wrapping_add` and `+` should be exactly the same, right?